FullStackMVCAPP/FullStackMVCAPP/DataContext/GOTContext.cs

```diff
@@ -6,9 +6,9 @@ namespace FullStackMVCAPP.DataContext
 public class GOTContext : DbContext 
 {
     
-    public DbSet<FullStackMVCAPP.Models.House> Houses { get; set; }
-    public DbSet<FullStackMVCAPP.Models.Castle> Castles { get; set; }
-    public DbSet<FullStackMVCAPP.Models.Character> Characters { get; set; }
+    public virtual DbSet<FullStackMVCAPP.Models.House> Houses { get; set; }
+    public virtual DbSet<FullStackMVCAPP.Models.Castle> Castles { get; set; }
+    public virtual DbSet<FullStackMVCAPP.Models.Character> Characters { get; set; }
```

Comment:
`FullStackMVCAPP.Models.Character` can be shorted to `Character` by importing as follows
```
using FullStackMVCAPP.Models;
```